src/query/src/sql.rs

```diff
@@ -65,6 +65,7 @@ const TABLES_COLUMN: &str = "Tables";
 const FIELD_COLUMN: &str = "Field";
 const TABLE_TYPE_COLUMN: &str = "Table_type";
 const COLUMN_NAME_COLUMN: &str = "Column";
+const COLUMN_GREPTIME_TYPE_COLUMN: &str = "Greptime_Type";
```

Comment:
`Greptime_Type` or `Greptime_type`? Because I saw `Table_type` at line 66

Reply:
Make sense, i'll change it.